components/messaging/ConversationCard.tsx

```diff
@@ -39,7 +39,7 @@ const ConversationCard: React.FC<ConversationCardProps> = ({
   currentConversationId,
 }) => {
   const isBreakpoint = useMediaQuery(1000);
-  const [imgSrc, setImgSrc] = useState(partnerAvatar);
+  const [imgSrc, setImgSrc] = useState(partnerAvatar || '/default-profile.png');
```

Comment:
watch out for this one! 

The `OR` operator is not what we'd want for this, it should be a `nullish coalescent operator` (`??`) to return the string '/default-profile.png' if `partnerAvatar` is`null`

_however_

what's really happening here is that on line 62 the `Image` component has an `onError` attribute that is intercepting the error and setting the `src` to the default image.

Right now the error shows up on the console because it's that error that's triggering the default avatar being displayed, but if you comment out the `onError` attribute you will see that we don't get any image anymore.

Unfortunately I don't think this issue is a frontend one, we should think of a more robust solution for our database really - ensuring images for our `dev` are persistent across separate environments but also injecting the default profile picture into the users' data if we see they don't have one. That way we would find the error once and clear it permanently and remove the weight from our frontend.

Until then, I think the component is handling it gracefully enough